dawn/test/integration-test/from-sir/TestComputeMaximumExtent.cpp

```diff
@@ -29,52 +28,25 @@ using namespace dawn;
 
 namespace {
 
-class TestComputeMaximumExtent : public ::testing::Test {
-  dawn::OptimizerContext::OptimizerContextOptions options_;
-  DiagnosticsEngine diagnostics_;
-  dawn::DawnCompiler compiler_;
-  std::unique_ptr<OptimizerContext> context_;
+std::shared_ptr<iir::StencilInstantiation> loadTest(const std::string& sirFilename) {
+  const std::string filename = TestEnvironment::path_ + "/" + sirFilename;
+  std::ifstream file(filename);
+  DAWN_ASSERT_MSG((file.good()), std::string("File " + filename + " does not exists").c_str());
 
-protected:
-  virtual void SetUp() {}
+  const std::string jsonstr((std::istreambuf_iterator<char>(file)),
+                            std::istreambuf_iterator<char>());
 
-  TestComputeMaximumExtent() {
-    context_ = std::make_unique<dawn::OptimizerContext>(diagnostics_, options_, nullptr);
-  }
+  auto sir = SIRSerializer::deserializeFromString(jsonstr, SIRSerializer::Format::Json);
+  // stage merger segfaults if stage reordering is not run beforehand
```

Comment:
this can't be. Groups should be individual. Can we get an issue?

Reply:
Looks like it is no longer the case. Eddie is changing the test anyway.